Juan Vazquez/exercise 1.py

```diff
@@ -0,0 +1,22 @@
+#Se utiliza <while true> para que la app se ejecute todo el tiempo para mejor comodidad al probarla.
+while True: 
+
+
+# Se agrega valor a la variable principal "number_to_multiply"
+   number_to_multiply = int(input("\n Hi! Please enter a number to get their square: "))
+
+   number_to_multiply = number_to_multiply + 1
+
+
+   if (number_to_multiply < 101):
+      for i  in range(number_to_multiply):
+         print(i * i) 
+      print("Your squared value is:", i * i)  
```

Comment:
Estas haciendo uso de la i afuera del for y eso no es correcto

Reply:
Además, por qué imprimes al final de nuevo la operación?

Reply:
Ok. Lo modifico.

La segunda impresión solo fue para resaltar el valor obtenido.

Reply:
Ahh va